Modules/CraftQueue/CraftQueue.lua

```diff
@@ -705,6 +707,29 @@ function CraftSim.CRAFTQ:GetRestockOptionsForRecipe(recipeID)
     return restockPerRecipeOptions[recipeID]
 end
 
+-- Function to determine the restock quantity for a given expected item.
+-- 
+-- This function retrieves the default restock amount from the database options.
+-- If a custom restock key for TSM (TradeSkillMaster) is specified, it attempts to get the restock amount
+-- using the TSM API based on the expected item's link. If a valid restock amount is found, it returns that value.
+-- Otherwise, it falls back to the default restock amount.
+--
+-- @param expectedItem The item for which the restock quantity is being determined.
+-- @return The restock quantity for the given expected item.

```

Comment:
syntax is --@param name type comment
and --@return type name comment

if u dont use luals in vscode yet i highly recommend it for code completion

Reply:
Ok, thanks for these informations